cmd/harmony/main.go

```diff
@@ -662,7 +661,7 @@ func setupConsensusAndNode(hc harmonyConfig, nodeConfig *nodeconfig.ConfigType)
 	} else if hc.DNSSync.LegacySyncing {
 		currentNode.SyncingPeerProvider = node.NewLegacySyncingPeerProvider(currentNode)
 	} else {
-		currentNode.SyncingPeerProvider = node.NewDNSSyncingPeerProvider(hc.DNSSync.Zone, legacysync.GetSyncingPort(strconv.Itoa(hc.DNSSync.Port)))
+		currentNode.SyncingPeerProvider = node.NewDNSSyncingPeerProvider(hc.DNSSync.Zone, strconv.Itoa(hc.DNSSync.Port))
```

Comment:
This line has already been fixed in v1.0.5 PR. I will do another pass to make sure every logic is expected.